### PR TITLE
s6t1 for review

### PR DESCRIPTION
**Pull request by EleanorRoberts**

### CODE REVIEW

File: SCSS/Imports/_museum.scss [hunk 1]
```diff
@@ -42,3 +42,18 @@
   color: #ffffff;
 }
 
+.pagination{
+  .btn {
+    margin: 5px;
```
Comment (jesshamby) [inline SCSS/Imports/_museum.scss]: not sure this is working? maybe try adding a class of m-1 to the buttons and make use of bootstrap :)

File: index.php [hunk 2]
```diff
@@ -34,6 +34,12 @@ function searchedValue(): string {
     }
 }
 
+// Pagination
+$pageNumber = $_GET['pageNumber'] ?? 1;
+$numberOfDinosPerPage = 10;
+$offset = ($pageNumber-1) * $numberOfDinosPerPage;
+$totalPages = 10;
```
Comment (jesshamby) [inline index.php:41]: Will this be a calculation in the next task rather than a set value?

**Review by jesshamby — COMMENTED** [review]
Looooooks good :) Just a small margin comment which may be fixed with bootstrap

File: index.php (hunk 2 above)
Comment (EleanorRoberts) [reply index.php:41]: Yes

**Pull request merged**